src/modal.js

```diff
@@ -0,0 +1,60 @@
+import Base from "./base.js";
+
+class Modal extends Base{
+  constructor(elt, config, onSubmit){
+    super(elt);
+    this.config = config;
+    this.onSubmit = onSubmit;
+    this._render();
+  }
+
+  _render(){
```

Comment:
There is a special syntax for private methods in ES6
Please, use it https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Private_class_fields